front/app/components/FilterSelector/index.tsx

```diff
@@ -207,7 +207,7 @@ const FilterSelector = ({
   };
 
   const handleKeyDown = (event: KeyboardEvent) => {
-    if (event.key === 'ArrowDown' && !opened) {
+    if (event.key === 'ArrowDown' && !opened && !multipleSelectionAllowed) {
```

Comment:
Add a comment to explain this, it is not super clear why you added this condition.